docs/proposals/2019-04-23 - Audio-Stream.md

```diff
@@ -0,0 +1,50 @@
+# Problem
+
+Audio stream files have ads stitched into the stream thus duration is different for those with ads as opposed to those without ads. I propose that we add duration to each stream element and a boolean to indicate whether or not an ad is stitched in.
+
+
+# Proposal
+
+Add duration and ad enabled boolean to each stream element inside of the streams array.
+
+
+## Duration
+
+Runtime of audio stream in milliseconds
+
+## Ad_enabled
```

Comment:
`ad_enabled` says to me that one _could_ put an ad in the stream, not that an ad is already in the stream. `contains_ad` or something like that seems clearer.